components/ProjectDetails.js

```diff
@@ -38,7 +64,11 @@ export default function ProjectDetails({ currentProject }) {
             <li key={step.id}>{step.desc}</li>
           ))}
         </ol>
+        <button onClick={() => handleDelete()}>Delete</button>
```

Comment:
There is no need to for creating an additional anonymous function.

```suggestion
        <button onClick={handleDelete}>Delete</button>
```

Reply:
👍🏾